### DISCUSSION TITLE
Guide for Overload Joystick Mapping w/Spaceorb & Orbotron 9001 - Solution

### DISCUSSION THREAD

**Discussion by gamer85** [thingotron/orb9k_circuitpython#3] [General]
Solution for those looking for SpaceOrb 360 joystick mapping with Overload. With the help of @Yoshimitsu#8541 (Discord), who created an Autocad SpaceOrb360 wireframe diagram, I was able to get the following optimal joystick mapping to work with Overload.
Also see Yoshi's Youtube Channel, esp the video at 
www.youtube.com/watch?v=voiENb-WXRU&t=18s

See Pictures "SpaceOrb Axes Diagram" & "Overload Controller Mapping" attached.  I prefer the "Least Torque Setting"

Turn Left - Twist the ball in the opposite direction of RZ arrow
Turn Right - Twist in the direction of RZ arrow
Pitch Up - Twist in the opposite direction of RX arrow (Up and over Top) 
Pitch Down - Twist in direction of RX arrow 
Roll Left – Twist in the opposite of RY arrow 
Roll Right - Twist in the direction of RZ arrow 
Move Forward - Push the ball forward in the opposite direction of the TY Arrow 
*Also Button D to minimize fatigue of having to push ball away from you
Move Back - Pull in the direction of the TY arrow (pull whole ball toward your body)
*Also Button F to minimize fatigue of having to pull ball toward you
Slide Left - Pull the ball straight across in the direction of the TX arrow 
Slide Right - Push straight across in the opposite direction of the TX arrow 
Slide Up - Pull the ball up off the body of controller in the opposite direction of TZ Slide Down - Push down toward the body of controller in the direction of TZ 
Button C - Primary Weapon 
Button E - Secondary Weapon

=
![SpaceOrb-axes Vertical](https://user-images.githubusercontent.com/20948864/174496203-db506831-2897-4ebc-8ba5-42fd148c392f.jpg)
![SpaceOrb-axes Horizontal](https://user-images.githubusercontent.com/20948864/174496205-b73701f3-6bef-40d8-a5c2-d58d6bf9091a.jpg)
![SpaceOrb-axes Least Torque](https://user-images.githubusercontent.com/20948864/174496206-d8c43ceb-22bf-4a7d-a596-e4ec51543746.jpg)

![Overload Controller Mapping](https://user-images.githubusercontent.com/20948864/172060660-f7d8b6b6-fa78-49ef-9b0b-7d10a5356e4d.jpg)

**Comment by petersem** [comment]
Super helpful, thank you

**Comment by shanselman** [comment]
This is PURE WIN
